pkg/plugin/work-in-progress/event_handler_unit_test.go

```diff
@@ -20,7 +20,8 @@ var _ = Describe("Work-in-progress Plugin features", func() {
 
 		DescribeTable("should recognize PR as work-in-progress if title starts with configured or default prefix",
 			func(title string) {
-				Expect(handler.IsWorkInProgress(title, wip.PluginConfiguration{Prefix: []string{`On Hold`}, Combine: true})).To(BeTrue())
+				state, _ := handler.HasWorkInProgressPrefix(title, wip.PluginConfiguration{Prefix: []string{`On Hold`}, Combine: true})
```

Comment:
could you please also verify (in all cases) the retrieved prefix - this way you verify that the [find](https://github.com/arquillian/ike-prow-plugins/pull/189/files#diff-11865c15f9f479c8ca04795d06b465acR182) works correctly